tools/exchange/src/main/scala/com/vesoft/nebula/tools/importer/config/Configs.scala

```diff
@@ -539,6 +542,31 @@ object Configs {
                                 config.getString("service"),
                                 config.getString("admin"),
                                 options)
+      case SourceCategory.HBASE =>
+        val fields: ListBuffer[String] = new ListBuffer[String]
```

Comment:
ditto

Reply:
the loop has been changed to 
`fields.append(config.getStringList("fields").asScala.map(field => field):_*)`